pandas/io/formats/style.py

```diff
@@ -21,13 +22,16 @@
 from pandas.core.dtypes.generic import ABCSeries
 
 import pandas as pd
+from pandas._typing import Axis, FrameOrSeries
 from pandas.api.types import is_dict_like, is_list_like
 import pandas.core.common as com
 from pandas.core.generic import _shared_docs
-from pandas.core.indexing import _maybe_numeric_slice, _non_reducing_slice
+from pandas.core.indexing import _IndexSlice, _maybe_numeric_slice, _non_reducing_slice
 
-jinja2 = import_optional_dependency("jinja2", extra="DataFrame.style requires jinja2.")
+ApplyArgs = Tuple[Callable[[FrameOrSeries], FrameOrSeries], Axis, Optional[_IndexSlice]]
+Kwargs = Dict[str, Any]
```

Comment:
Hmm not sure about this; I think would be good to limit the amount of global aliases in use. 

Do you see a general use for this outside of the ninja example below?

Reply:
no, but alias used for readability of L130

if we don't use aliases, what's your preference for formatting?

https://github.com/python/mypy/issues/4511

Reply:
i've retained the aliases for readability since "Type aliases are useful for simplifying complex type signatures" https://docs.python.org/3/library/typing.html#type-aliases

but moved them to just before the class where they are used. This pattern is used in https://github.com/python/typeshed/blob/master/stdlib/2and3/codecs.pyi